src/components/PasswordInput.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
```

Comment:
You already have `Login` module (it should be not a file, but a folder). Move all the logic that's related _only_ to this module to that folder. This component probably should be re-used for some generic solution. Is it possible to re-write it? Like If you pass `password`, `setPassword` and `secureTextEntry` from the parent, you can rename the component to `Input` and reuse it everywhere.